### PR TITLE
Only set Authorization header if Token exist

### PR DESCRIPTION
**Pull request by aslakknutsen**
* Call all API's with Authorization header

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

File: src/app/work-item/work-item.service.ts [hunk 1]
```diff
@@ -17,7 +17,9 @@ export class WorkItemService {
   constructor(private http: Http,
               private logger: Logger,
               private auth: AuthenticationService) {    
-    this.headers.append('Authorization', 'Bearer ' + this.auth.getToken());
+    if(this.auth.getToken() != null) {
```
Comment (aslakknutsen) [inline src/app/work-item/work-item.service.ts]: maybe this should call this.auth.isLoffedIn() instead of checking token status?

Comment (joshuawilson) [inline src/app/work-item/work-item.service.ts]: @aslakknutsen do you want to change it before I merge it?

Comment (joshuawilson) [inline src/app/work-item/work-item.service.ts]: @aslakknutsen I merged it, if you want to make that change please send a new PR once this branch is in master.

**Pull request merged**